test/cases/image_tests.sh

```diff
@@ -57,6 +58,17 @@ get_test_cases () {
         else
             SKIP_CASES=("${SKIP_OSTREE[@]}")
         fi
+
+        # skip container test if running on old osbuild version
+        # b/c manifests have been modified to include the "source" option
+        # which is not yet supported on 8.8 and 9.2 downstream
```

Comment:
Not a big issue, but the actual new options are `manifest-lists` (for skopeo inputs) and the new `skopeo-index` source.
It might save someone some trouble in the future if they went looking for the reason for this skip.

Reply:
Updated.